modules/standard/IO.chpl

```diff
@@ -1096,10 +1096,10 @@ class QioPluginFile {
   }
 
   /* Returns the path to an open file.
-     The caller has the responsibility to free the returned c_string.
+     The caller has the responsibility to free the returned c_ptrConst(c_char).
    */
   // TODO: use Chapel strings for this, one day
-  proc getpath(out path:c_string, out len:int(64)):errorCode {
+  proc getpath(out path:c_ptr(uint(8)), out len:int(64)):errorCode {
```

Comment:
Probably worth a comment about why it is uint(8). Also adjust comment on 1099 to not be wrong.